COMMITTERS.csv

```diff
@@ -0,0 +1,11 @@
+Name, Email, Github ID
+Alan Chin, IBM, akchinSTC 
```

Comment:
The second column is labeled `Email` but appears to be `Affiliation`.

This file could use a newline at EOF.